release/el7/Dockerfile

```diff
@@ -1,30 +1,35 @@
 FROM centos:7
 MAINTAINER Andrew Bauer <zonexpertconsulting@outlook.com>
 
-# Enable the EPEL repo. The repo package is part of centos base so no need fetch it.
-RUN yum -y install epel-release
-
-# Fetch and enable the RPMFusion repo
-RUN yum -y localinstall --nogpgcheck https://download1.rpmfusion.org/free/el/rpmfusion-free-release-7.noarch.rpm
-
-# Install the latest *release* of zoneminder
-RUN yum -y install zoneminder mariadb-server mod_ssl zip file
+# Get the entrypoint script
+ADD https://raw.githubusercontent.com/ZoneMinder/zmdockerfiles/master/utils/entrypoint.sh /usr/local/bin/
 
-# Set our volumes before we attempt to configure apache
+## Dependencies layer
+RUN \
+  # Enable the EPEL repo. The repo package is part of centos base so no need fetch it.
+  yum -y install epel-release \
+  # Fetch and enable the RPMFusion repo
+  && yum -y localinstall --nogpgcheck https://download1.rpmfusion.org/free/el/rpmfusion-free-release-7.noarch.rpm \
+  # Install dependencies
+  && yum -y install mariadb-server mod_ssl zip file \
+  # Clean up yum cache
+  && yum clean all \
```

Comment:
Do we need to yum clean here, since we are also yum clean'ing after we yum install zoneminder?

Reply:
I haven't tested the result of _not_ having it here... but my gut says it should be there. Without it, the first `RUN` layer will still have the yum cache, and will be larger than if we didn't run it. Removing it in subsequent layers (`RUN`s) will not remove it from the layer created in the first `RUN`.